ci-operator/step-registry/ipi/conf/etcd/on-ramfs/ipi-conf-etcd-on-ramfs-ref.yaml

```diff
@@ -15,5 +15,12 @@ ref:
     - name: IGNITIONVERSION
       default: "3.1.0"
       documentation: ignition version to use for mco.
+    - name: USE_RAMFS
+      default: "true"
+      documentation: |-
```

Comment:
so now we also need to set it to false on the upgrade jobs ?

Reply:
Yes.  How else would you go about coding it?

Reply:
this is fine - i was just noting that in this same PR we also want to do that - set use_ramfs to false for the upgrade jobs

Reply:
Cool.  I wanted agreement before I altered the upgrade jobs. :) So then I wouldn't have to undo a bunch of stuff...

Reply:
And hopefully someone from openstack-approvers agrees as well.

Reply:
I think this is perfectly sane, but yes if we can get a ACK from ovirt and OpenStack approvers that would be good. there is one job that they run that is an upgrade job. IMHO it should be broken, since it is affected by this topology:
`periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-4.10-upgrade-from-eus-4.8-openstack-ipi-f28`

Reply:
I don't see it in sippy, so maybe it doesn't run 🤷

Reply:
I feel we shouldn't set this option as true because we need to increase the memory as well for the nodes to accommodate that additional capacity for the etcd pod